README.md

```diff
@@ -2,30 +2,31 @@
 
 [![Build Status](https://travis-ci.org/geerlingguy/ansible-role-certbot.svg?branch=master)](https://travis-ci.org/geerlingguy/ansible-role-certbot)
 
-Installs Certbot (for Let's Encrypt) for RHEL/CentOS or Debian/Ubuntu.
+Installs and configures Certbot (for Let's Encrypt).
 
 ## Requirements
 
-Certbot requires Git to be installed. You can install Git using the `geerlingguy.git` role.
+Certbot requires Git to be installed if one wants to install Certbot from Git repository instead of package management. You can install Git using the `geerlingguy.git` role.
 
 ## Role Variables
 
+    certbot_from_git: no
```

Comment:
I'd rather call this variable `certbot_install_from_source` so it's consistent with my other roles.